test/views/declaration/BorderTransportViewSpec.scala

```diff
@@ -139,15 +139,15 @@ class BorderTransportViewSpec extends UnitViewSpec with ExportsTestData with Stu
         )
       }
     }
+  }
 
   private def createView(mode: Mode = Mode.Normal, form: Form[BorderTransport] = form, request: JourneyRequest[_] = journeyRequest()): Document =
     page(mode, form)(request, realMessages)
 
-  "TransportDetails View" when {
-    "we are on Standard journey" should {
-      val requrestOnStandard = journeyRequest(DeclarationType.STANDARD)
+  "TransportDetails View" must {
 
-      val view = createView(request = requrestOnStandard)
+    onStandard { request =>
```

Comment:
I'm not a fan of this style

I'd prefer if we just added a method to request e.g. `request.asStandard` or something

Reply:
I'm open to seeing what the team thinks though

Reply:
I'm not sure what's going on here.  What's "onStandard" etc?

Reply:
IT would be good to explain to us about this approach.

Reply:
This is support method on ViewSpec that would add prefix for registration test case part of `onAllJourneys` and `onJourney`